packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -130,7 +131,10 @@ class WebView extends StatefulWidget {
   ///
   /// A null value is equivalent to an empty set.
   final Set<JavascriptChannel> javascriptChannels;
-
+  
+  /// Enable/Disable zoom.
```

Comment:
Is there a way to do the same on iOS?
If yes we should support that as well, if no the comment should mention that this is a no-op on iOS.

Reply:
at the moment I have no way to test the ios environment. I will modify the comment in on-op on ios